cmake/irrlichtmtConfig.cmake

```diff
@@ -0,0 +1,8 @@
+get_filename_component(IrrlichtMt_CMAKE_DIR "${CMAKE_CURRENT_LIST_DIR}" PATH)
+include(CMakeFindDependencyMacro)
+
+find_dependency(ZLIB REQUIRED)
```

Comment:
why is zlib here but not the rest?

is this needed at all?
looking at the generated `irrlichtmtTargets.cmake` there's this, which already specifies dependent libraries:
<pre>
set_target_properties(irrlichtmt::IrrlichtMt PROPERTIES
  INTERFACE_LINK_LIBRARIES "/usr/lib/x86_64-linux-gnu/libz.so;/usr/lib/x86_64-linux-gnu/libjpeg.so;/usr/lib/x86_64-linux-gnu/libpng.so;/usr/lib/x86_64-linux-gnu/libGL.so;/usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXxf86vm.so"
)
</pre>

Reply:
Thanks for bringing that to my attention. That was in an example I used as a reference, so I added one to remind me to research it and figure out whether it's needed. The explanation wasn't very clear.

Reply:
Ah. It looks like those are needed if the Targets file references other targets. In Irrlicht's case, it's unecessary, because Irrlicht doesn't link zlib etc. as targets (although it should). I'll remove the dependency for now, and then I can re-add it in a separate PR to link those libraries as targets.